src/main/java/racing/winner/domain/CarBuilder.java

```diff
@@ -0,0 +1,31 @@
+package racing.winner.domain;
+
+import racing.winner.resolver.CarDto.Request;
+import racing.winner.resolver.CarDto.Response;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.collectingAndThen;
+import static java.util.stream.Collectors.toList;
+
+public class CarBuilder {
+
+    public static List<Car> toCar(Request.GameCreation creation) {
+        List<String> names = creation.getCarNames();
+        return names.stream()
+                .map(Car::new)
+                .limit(names.size())
+                .collect(collectingAndThen(toList(), Collections::unmodifiableList));
+    }
```

Comment:
limit 을 해줄 필요가 없을 것 같아요 😄 
```suggestion
    public static List<Car> toCar(Request.GameCreation creation) {
        List<String> names = creation.getCarNames();
        return names.stream()
                .map(Car::new)
                .collect(collectingAndThen(toList(), Collections::unmodifiableList));
    }
```

Reply:
Stream 을 이용하여 generate 시 에는 limit 이 필요하고, 이곳에서는 필요하지 않은 것이군요
코드 반영하고 좀더 알아보겠습니다 감사합니다